packages/expo-modules-core/ios/NativeModulesProxy/EXNativeModulesProxy.m

```diff
@@ -28,6 +29,13 @@ - (void)registerAdditionalModuleClasses:(NSArray<Class> *)modules;
 
 @end
 
+@interface RCTComponentData (EXRCTComponentData)
```

Comment:
I'm not sure if we want to prefix it or not 🤷 
And maybe the name isn't the best 🤔  What do you think?

Reply:
Let's go with this 👇, the category name is usually a name of the class that needs it 🙂 
```suggestion
@interface RCTComponentData (EXNativeModulesProxy)
```